tools/ctl/ctl.go

```diff
@@ -444,6 +445,34 @@ var listRuns = &cobra.Command{
 	},
 }
 
+var infer = &cobra.Command{
+	Use:   "infer --ecosystem <ecosystem> --package <name> --version <version> [--artifact <name>]",
```

Comment:
Would it be possible to support both remote and local variants here? Would be nice to be able to dispatch to the hosted version, too

Reply:
Added